android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -1935,6 +1943,10 @@ public void onTabDocumentLoaded(String tag) {
             }
         }
 
+        if (mIsBase64 && mDocumentUri.getPath() != null) {
+            mTempFiles.add(new File(mDocumentUri.getPath()));
```

Comment:
i think this can be moved directly under "ReactUtils.getUri(getContext(), path, mIsBase64, mBase64Extension);" because that's when the temp file is created